src/content/en/fundamentals/instant-and-offline/get-started/graphicalcontent-4.md

```diff
@@ -0,0 +1,284 @@
+saving it to the version seen above at 115k. That's a 52% reduction, and the photo now 
+focuses on the important parts (yes, the doghouse is important) at half the file size. 
+All the required information is there, it's just more efficiently presented.
+
+###Reduce image quality
+
+Of course, we want our images to look as good as possible, within reason, so that means saving 
+JPGs at 100% quality, right? Not necessarily. In most (which is to say nearly all) cases, 
+you can reduce the JPG quality, and thus the file size, without suffering any visible quality 
+difference.
+
+Once you've determined the size that an image should really be, how do you get it to that size 
+but still retain the quality and characteristics it needs? The first thing you need is a good 
+image manipulation tool. 
+[ResizeIt](https://itunes.apple.com/us/app/resizeit/id416280139?mt=12) 
+is a Mac-only desktop product lets you change the size of 
+multiple images simultaneously, and can convert file formats at the same time. 
+
+Have a look at the following photograph-type JPGs, where the quality is progressively 
+reduced (using the popular Windows desktop product 
+[paint.net](https://www.getmaint.net)), 
+and note the decreasing file sizes. A quick calculation reveals that the size difference 
+between the 100% quality image and the 25% quality image is 90%! But can you actually 
+see the difference? Perhaps more importantly, even if you *can* see any difference, 
+is it significant enough to warrant a larger file and thus a slower download?
+
+![burger100.jpg](images/image_405.jpg)
+*burger100.jpg -- The original image at 100%, 227k*
+
+![paint.net screenshot](images/image_400.png)
+*Using paint.net to save at lower quality*
+
+![burger80.jpg](images/image_406.jpg)
+*burger80.jpg -- 80%, 60k*
+
+![burger50.jpg](images/image_407.jpg)
+*burger50.jpg -- 50%, 34k*
+
+![burger25.jpg](images/image_408.jpg)
+*burger25.jpg -- 25%, 20k*
+
+The takeaway is obvious: definitely experiment with lower-quality JPGs to see how low you can 
+go before seeing a difference, and then use the smallest one that retains the photo's clarity. 
+
+Additional image manipulation tools:
+
+- http://www.picresize.com/: One of several good browser-based tools that gives you lots of 
+options for cropping, rotating, resizing, adding effects to, and converting images.
+- https://www.gimp.org/: This ever-popular cross-platform tool just gets better with age. 
+Powerful and flexible, Gimp lets you perform a wide variety of image manipulation tasks 
+including, of course, resizing.
+
+This is just a smattering of available tools. We aren't here for a comparative review of 
+image editors, although CrazyEgg has a comprehensive article here: 
+
+[https://www.crazyegg.com/blog/image-editing-tools/](https://www.crazyegg.com/blog/image-editing-tools/)
+
+##Compress Images
+
+PNG and JPG images can be squashed down even more using a compression tool. For example, 
```

Comment:
Suggestion to introduce the image that gets place in TinyPNG, and returns compression results.